lib/IntegrationEvent/Implementation/AWS-SNS-SQS/RedisHelper.ts

```diff
@@ -0,0 +1,76 @@
+import { IntegrationEvent } from "../../IntegrationEvent";
+import { WrappedNodeRedisClient } from "handy-redis";
+
+export class RedisHelper {
+  public static async storeEventData(options: {
+    event: IntegrationEvent;
+    RedisClient: WrappedNodeRedisClient;
+  }): Promise<void> {
+    if (options.RedisClient && options.RedisClient.nodeRedis.connected && options.event && options.event.queueId) {
+      try {
+        const hashKey = `{${options.event.queueId}}_data`;
```

Comment:
Even though this is not official, Redis documentations and tutorials often use colon `:` as a separator. And generally the category names are in front. Can we follow this convention instead?

So data:queueID and queue:queueID for example.

Keys like user:12345:device:abc is more intuitive to read than abc:device:12345:user.

Reply:
Makes sense. I will update it.

Reply:
This has been updated in [885425b](https://github.com/Polarrco/mediator/pull/1/commits/885425b47478d73a6e3efff779e6c43722cdee6e)